docs/jsdocs.html

```diff
@@ -7259,15 +7259,15 @@ <h4>January 9th, 2022</h4>
 							<div class="spoiler">
 								<div class="settingsDiv">
 									<p><span class="hl">Prototype</span></p>
-									<p><em>MakeItem( socket, player, itemNum );</em></p>
+									<p><em>MakeItem( socket, player, itemNum, hueNum);</em></p>
 								</div>
 								<div class="settingsDiv">
 									<p><span class="hl">Purpose</span></p>
 									<p>Player makes the item itemNum. itemNum points to a create.dfn (SECTION ITEM itemNum) entry. Socket can be -1</p>
```

Comment:
What does it mean if socket is -1?  Does that mean it's not a Player but NPC, or just not people being told about it?
Adding some info re hue wouldn't hurt either, particularly if there's an offset from 0 issue